### ISSUE TITLE
gssapi error on require 'kafka`

### ISSUE THREAD

**Issue by ale7714** [zendesk/ruby-kafka#389]
* Version of Ruby: 2.2.2p95
* Version of Kafka: 0.9
* Version of ruby-kafka: master

When using `ruby-kafka` master branch and trying to run the service, it fails on `require "kafka"`. The error looks something like:
```
/usr/lib/ruby/gems/2.2.0/gems/ffi-1.9.18/lib/ffi/library.rb:147:in `block in ffi_lib': Could not open library 'libgssapi_krb5.so.2': Error loading shared library libgssapi_krb5.so.2: No such file or directory. (LoadError)
```

We're not using `sasl_gssapi_auth` and that's why we don't have any dependency for gssapi installed.

**Comment by dasch** [comment]
Argh, that's what I feared. I'd love a PR that conditionally and lazily requires the gssapi library only when needed.

**Comment by ale7714** [comment]
That sounds like a good plan, I'll look into it during the weekend

**Comment by ale7714** [comment]
@dasch i opened this PR https://github.com/zendesk/ruby-kafka/pull/391 to address this. Let me know what you think.

**Comment by dasch** [comment]
Fixed by #391.
